openedx_tagging/core/tagging/import_export/actions.py

```diff
@@ -409,8 +406,7 @@ def execute(self) -> None:
         Delete a tag
         """
         try:
-            taxonomy_tag = self._get_tag()
-            taxonomy_tag.delete()
+            self._get_tag().delete()
```

Comment:
nit: similar to the above, if this had any performance improvement or just code style changes?

Reply:
Same as above